starters/qwik-graphql-tailwind/src/routes/counter/counter.test.tsx

```diff
@@ -10,16 +9,62 @@ describe('Counter component', function () {
   });
 
   it('should render', async () => {
-    const fixture = new ElementFixture();
+    const { render, screen } = await createDOM();
 
-    await render(fixture.host, <Counter />);
+    await render(<Counter />);
 
-    expect(fixture.host.querySelectorAll('button').length).toBe(3);
-    expect(fixture.host.querySelector('h1')?.textContent).toBe('0');
+    expect(screen.querySelectorAll('button').length).toBe(3);
+    expect(screen.querySelector('h1')?.textContent).toBe('0');
   });
 
   it('should increment', async () => {
-    // Interaction doesn't work at the moment because we don't have a way to trigger an event in the test environment.
-    // See https://github.com/BuilderIO/qwik/discussions/1801 for more details.
+    const { render, screen, userEvent } = await createDOM();
+
+    await render(<Counter />);
+
+    const plusButton = screen.querySelectorAll('button')[1];
+    // eslint-disable-next-line qwik/no-use-after-await
```

Comment:
This is clearly a bug in the `qwik/no-use-after-await` rule in `eslint-plugin-qwik` - they just check whether the method starts with "use". The rule seems to be deprecated, but if you feel like it, you could create an issue: https://github.com/BuilderIO/qwik/issues